language/German/strings.po

```diff
@@ -2249,7 +2266,7 @@ msgstr "Vorschau Illustrationen"
 #: src/frontend/mame/ui/dirmenu.cpp:69
 msgctxt "path-option"
 msgid "Select"
-msgstr "Auswahl"
+msgstr "Charakterauswahl"
```

Comment:
If you’re changing it here, you should probably change it on line 3081 (`msgctxt "selmenu-artwork"`) as well.

Reply:
Fixed there as well